core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/RepairStateImpl.java

```diff
@@ -194,10 +192,9 @@ private long partiallyRepairedTableRepairedAt(final long maxRepairedAt, final Re
         {
             next -= old.getEstimatedRepairTime();
         }
-        if (LOG.isInfoEnabled())
+        if (LOG.isDebugEnabled())
```

Comment:
Maybe this can be moved to line 189-190, as everything thereafter is only done for logging purposes. Similarly to previous method.